Sources/Extensions/UIKit/UIEdgeInsetsExtensions.swift

```diff
@@ -44,6 +44,60 @@ extension UIEdgeInsets {
     public init(horizontal: CGFloat, vertical: CGFloat) {
         self.init(top: vertical/2, left: horizontal/2, bottom: vertical/2, right: horizontal/2)
     }
+
+    /// SwifterSwift: Creates an `UIEdgeInsets` based on current value and top offset.
+    ///
+    /// - Parameters:
+    ///   - top: Offset to be applied in to the top edge.
+    /// - Returns: UIEdgeInsets offset with given offset.
+    public func insetBy(top: CGFloat) -> UIEdgeInsets {
+        return UIEdgeInsets(top: self.top + top, left: left, bottom: bottom, right: right)
+    }
+
+    /// SwifterSwift: Creates an `UIEdgeInsets` based on current value and left offset.
+    ///
+    /// - Parameters:
+    ///   - top: Offset to be applied in to the left edge.
+    /// - Returns: UIEdgeInsets offset with given offset.
+    public func insetBy(left: CGFloat) -> UIEdgeInsets {
+        return UIEdgeInsets(top: top, left: self.left + left, bottom: bottom, right: right)
+    }
+
+    /// SwifterSwift: Creates an `UIEdgeInsets` based on current value and bottom offset.
+    ///
+    /// - Parameters:
+    ///   - top: Offset to be applied in to the bottom edge.
+    /// - Returns: UIEdgeInsets offset with given offset.
+    public func insetBy(bottom: CGFloat) -> UIEdgeInsets {
+        return UIEdgeInsets(top: top, left: left, bottom: self.bottom + bottom, right: right)
+    }
+
+    /// SwifterSwift: Creates an `UIEdgeInsets` based on current value and right offset.
+    ///
+    /// - Parameters:
+    ///   - top: Offset to be applied in to the right edge.
```

Comment:
@VincentSit  All the docs are with parameter `top` 😆

Reply:
Whoops. Good catch, thanks. Fixed. :joy: